lib/engine/game/g_1828.rb

```diff
@@ -122,6 +122,17 @@ def init_stock_market
         sm
       end
 
+      def init_tiles
+        tiles = super
+
+        tiles.find { |tile| tile.name == '53' }.label = 'Ba'
```

Comment:
is there only one tile of this type? if not you could do

Reply:
Yes, only one of each.